dkc/core/tests/test_metadata.py

```diff
@@ -22,6 +22,7 @@ def test_invalid_metadata(metadata, folder_factory):
     assert err.value.error_dict['user_metadata'][0].message == 'Must be a JSON Object.'
 
 
+@pytest.mark.xfail
```

Comment:
This failure needs to be addressed, probably by making this a database test, and manually saving and adding a `Collection` to the `Folder` before attempting the `full_clean`.

Other tests that call `full_clean` are actually encountering this error too, though they are always calling `full_clean` within a `ValidationError` capturing context, so the error goes unnoticed. To be as strict as possible in what errors we are trying to trigger in those test cases, we should perhaps add similar workarounds theres.